spec/factories/bars.rb

```diff
@@ -0,0 +1,7 @@
+FactoryGirl.define do
+  factory :bar do
+    name "Game On"
```

Comment:
You might want to sequence the name so they're unique. `sequence(:name) { |n| "Game On #{n}" }`